src/vs/workbench/contrib/terminal/browser/terminalInstance.ts

```diff
@@ -976,6 +976,11 @@ export class TerminalInstance extends Disposable implements ITerminalInstance {
 				return false;
 			}
 
+			if (event.key === 'Tab' && event.shiftKey) {
```

Comment:
```suggestion
			// Prevent default when shift+tab is being sent to the terminal to avoid it bubbling up
			// and changing focus https://github.com/microsoft/vscode/issues/188329
			if (event.key === 'Tab' && event.shiftKey) {
```